src/util/std_expr.h

```diff
@@ -3444,10 +3089,9 @@ inline void validate_expr(const if_exprt &value)
 }
 
 
-/*! \brief Operator to update elements in structs and arrays
-    \remark This expression will eventually be replaced by separate
-            array and struct update operators.
-*/
+/// \brief Operator to update elements in structs and arrays
+///    \remark This expression will eventually be replaced by separate
```

Comment:
Some of these have strange indent.. Not sure if it's showing in doc too, and if it's worth going over.